src/Bundler.js

```diff
@@ -207,7 +207,8 @@ class Bundler extends EventEmitter {
         });
       });
     }
-
+    this.emit('bundlestart', this.entryFiles)
```

Comment:
We currently have a `buildEnd` event when `Bundler` is done, it would be more consistent to name this one `buildStart`

Reply:
Definitely makes sense